{"cells":[{"cell_type":"markdown","source":["# selenium을 활용한 웹 스크래핑 실전!"],"metadata":{"id":"-4DVVyn9IvDO"}},{"cell_type":"markdown","metadata":{"id":"FD4KMgxOMfkz"},"source":["### 참조: https://aytekin.tistory.com/48\n","\n","### https://book.coalastudy.com/data_crawling/week3/stage3 를 대폭 수정함"]},{"cell_type":"code","execution_count":1,"metadata":{"id":"1wTs8-2tP8Wy","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1711495542515,"user_tz":-540,"elapsed":41360,"user":{"displayName":"KyungTae Lim","userId":"06695996952901489598"}},"outputId":"d13dbf76-7f97-424b-a81b-f41d683cdd1b"},"outputs":[{"output_type":"stream","name":"stdout","text":["Collecting selenium\n","  Downloading selenium-4.18.1-py3-none-any.whl (10.0 MB)\n","\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m10.0/10.0 MB\u001b[0m \u001b[31m15.4 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n","\u001b[?25hRequirement already satisfied: urllib3[socks]<3,>=1.26 in /usr/local/lib/python3.10/dist-packages (from selenium) (2.0.7)\n","Collecting trio~=0.17 (from selenium)\n","  Downloading trio-0.25.0-py3-none-any.whl (467 kB)\n","\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m467.2/467.2 kB\u001b[0m \u001b[31m32.8 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n","\u001b[?25hCollecting trio-websocket~=0.9 (from selenium)\n","  Downloading trio_websocket-0.11.1-py3-none-any.whl (17 kB)\n","Requirement already satisfied: certifi>=2021.10.8 in /usr/local/lib/python3.10/dist-packages (from selenium) (2024.2.2)\n","Requirement already satisfied: typing_extensions>=4.9.0 in /usr/local/lib/python3.10/dist-packages (from selenium) (4.10.0)\n","Requirement already satisfied: attrs>=23.2.0 in /usr/local/lib/python3.10/dist-packages (from trio~=0.17->selenium) (23.2.0)\n","Requirement already satisfied: sortedcontainers in /usr/local/lib/python3.10/dist-packages (from trio~=0.17->selenium) (2.4.0)\n","Requirement already satisfied: idna in /usr/local/lib/python3.10/dist-packages (from trio~=0.17->selenium) (3.6)\n","Collecting outcome (from trio~=0.17->selenium)\n","  Downloading outcome-1.3.0.post0-py2.py3-none-any.whl (10 kB)\n","Requirement already satisfied: sniffio>=1.3.0 in /usr/local/lib/python3.10/dist-packages (from trio~=0.17->selenium) (1.3.1)\n","Requirement already satisfied: exceptiongroup in /usr/local/lib/python3.10/dist-packages (from trio~=0.17->selenium) (1.2.0)\n","Collecting wsproto>=0.14 (from trio-websocket~=0.9->selenium)\n","  Downloading wsproto-1.2.0-py3-none-any.whl (24 kB)\n","Requirement already satisfied: pysocks!=1.5.7,<2.0,>=1.5.6 in /usr/local/lib/python3.10/dist-packages (from urllib3[socks]<3,>=1.26->selenium) (1.7.1)\n","Collecting h11<1,>=0.9.0 (from wsproto>=0.14->trio-websocket~=0.9->selenium)\n","  Downloading h11-0.14.0-py3-none-any.whl (58 kB)\n","\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m58.3/58.3 kB\u001b[0m \u001b[31m5.7 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n","\u001b[?25hInstalling collected packages: outcome, h11, wsproto, trio, trio-websocket, selenium\n","Successfully installed h11-0.14.0 outcome-1.3.0.post0 selenium-4.18.1 trio-0.25.0 trio-websocket-0.11.1 wsproto-1.2.0\n","Get:1 https://cloud.r-project.org/bin/linux/ubuntu jammy-cran40/ InRelease [3,626 B]\n","Hit:2 http://archive.ubuntu.com/ubuntu jammy InRelease\n","Hit:3 https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/x86_64  InRelease\n","Get:4 http://archive.ubuntu.com/ubuntu jammy-updates InRelease [119 kB]\n","Get:5 http://security.ubuntu.com/ubuntu jammy-security InRelease [110 kB]\n","Hit:6 http://archive.ubuntu.com/ubuntu jammy-backports InRelease\n","Hit:7 https://ppa.launchpadcontent.net/c2d4u.team/c2d4u4.0+/ubuntu jammy InRelease\n","Hit:8 https://ppa.launchpadcontent.net/deadsnakes/ppa/ubuntu jammy InRelease\n","Get:9 http://archive.ubuntu.com/ubuntu jammy-updates/universe amd64 Packages [1,356 kB]\n","Hit:10 https://ppa.launchpadcontent.net/graphics-drivers/ppa/ubuntu jammy InRelease\n","Get:11 http://security.ubuntu.com/ubuntu jammy-security/restricted amd64 Packages [2,048 kB]\n","Hit:12 https://ppa.launchpadcontent.net/ubuntugis/ppa/ubuntu jammy InRelease\n","Get:13 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 Packages [1,903 kB]\n","Get:14 http://security.ubuntu.com/ubuntu jammy-security/universe amd64 Packages [1,080 kB]\n","Get:15 http://security.ubuntu.com/ubuntu jammy-security/main amd64 Packages [1,624 kB]\n","Fetched 8,243 kB in 2s (4,358 kB/s)\n","Reading package lists... Done\n","Reading package lists... Done\n","Building dependency tree... Done\n","Reading state information... Done\n","The following additional packages will be installed:\n","  apparmor chromium-browser libfuse3-3 liblzo2-2 libudev1 snapd squashfs-tools systemd-hwe-hwdb\n","  udev\n","Suggested packages:\n","  apparmor-profiles-extra apparmor-utils fuse3 zenity | kdialog\n","The following NEW packages will be installed:\n","  apparmor chromium-browser chromium-chromedriver libfuse3-3 liblzo2-2 snapd squashfs-tools\n","  systemd-hwe-hwdb udev\n","The following packages will be upgraded:\n","  libudev1\n","1 upgraded, 9 newly installed, 0 to remove and 38 not upgraded.\n","Need to get 26.4 MB of archives.\n","After this operation, 116 MB of additional disk space will be used.\n","Get:1 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 apparmor amd64 3.0.4-2ubuntu2.3 [595 kB]\n","Get:2 http://archive.ubuntu.com/ubuntu jammy/main amd64 liblzo2-2 amd64 2.10-2build3 [53.7 kB]\n","Get:3 http://archive.ubuntu.com/ubuntu jammy/main amd64 squashfs-tools amd64 1:4.5-3build1 [159 kB]\n","Get:4 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 libudev1 amd64 249.11-0ubuntu3.12 [78.2 kB]\n","Get:5 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 udev amd64 249.11-0ubuntu3.12 [1,557 kB]\n","Get:6 http://archive.ubuntu.com/ubuntu jammy/main amd64 libfuse3-3 amd64 3.10.5-1build1 [81.2 kB]\n","Get:7 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 snapd amd64 2.58+22.04.1 [23.8 MB]\n","Get:8 http://archive.ubuntu.com/ubuntu jammy-updates/universe amd64 chromium-browser amd64 1:85.0.4183.83-0ubuntu2.22.04.1 [49.2 kB]\n","Get:9 http://archive.ubuntu.com/ubuntu jammy-updates/universe amd64 chromium-chromedriver amd64 1:85.0.4183.83-0ubuntu2.22.04.1 [2,308 B]\n","Get:10 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 systemd-hwe-hwdb all 249.11.5 [3,228 B]\n","Fetched 26.4 MB in 1s (18.5 MB/s)\n","Preconfiguring packages ...\n","Selecting previously unselected package apparmor.\n","(Reading database ... 121753 files and directories currently installed.)\n","Preparing to unpack .../apparmor_3.0.4-2ubuntu2.3_amd64.deb ...\n","Unpacking apparmor (3.0.4-2ubuntu2.3) ...\n","Selecting previously unselected package liblzo2-2:amd64.\n","Preparing to unpack .../liblzo2-2_2.10-2build3_amd64.deb ...\n","Unpacking liblzo2-2:amd64 (2.10-2build3) ...\n","Selecting previously unselected package squashfs-tools.\n","Preparing to unpack .../squashfs-tools_1%3a4.5-3build1_amd64.deb ...\n","Unpacking squashfs-tools (1:4.5-3build1) ...\n","Preparing to unpack .../libudev1_249.11-0ubuntu3.12_amd64.deb ...\n","Unpacking libudev1:amd64 (249.11-0ubuntu3.12) over (249.11-0ubuntu3.10) ...\n","Setting up libudev1:amd64 (249.11-0ubuntu3.12) ...\n","Selecting previously unselected package udev.\n","(Reading database ... 121961 files and directories currently installed.)\n","Preparing to unpack .../udev_249.11-0ubuntu3.12_amd64.deb ...\n","Unpacking udev (249.11-0ubuntu3.12) ...\n","Selecting previously unselected package libfuse3-3:amd64.\n","Preparing to unpack .../libfuse3-3_3.10.5-1build1_amd64.deb ...\n","Unpacking libfuse3-3:amd64 (3.10.5-1build1) ...\n","Selecting previously unselected package snapd.\n","Preparing to unpack .../snapd_2.58+22.04.1_amd64.deb ...\n","Unpacking snapd (2.58+22.04.1) ...\n","Setting up apparmor (3.0.4-2ubuntu2.3) ...\n","Created symlink /etc/systemd/system/sysinit.target.wants/apparmor.service → /lib/systemd/system/apparmor.service.\n","Setting up liblzo2-2:amd64 (2.10-2build3) ...\n","Setting up squashfs-tools (1:4.5-3build1) ...\n","Setting up udev (249.11-0ubuntu3.12) ...\n","invoke-rc.d: could not determine current runlevel\n","invoke-rc.d: policy-rc.d denied execution of start.\n","Setting up libfuse3-3:amd64 (3.10.5-1build1) ...\n","Setting up snapd (2.58+22.04.1) ...\n","Created symlink /etc/systemd/system/multi-user.target.wants/snapd.aa-prompt-listener.service → /lib/systemd/system/snapd.aa-prompt-listener.service.\n","Created symlink /etc/systemd/system/multi-user.target.wants/snapd.apparmor.service → /lib/systemd/system/snapd.apparmor.service.\n","Created symlink /etc/systemd/system/multi-user.target.wants/snapd.autoimport.service → /lib/systemd/system/snapd.autoimport.service.\n","Created symlink /etc/systemd/system/multi-user.target.wants/snapd.core-fixup.service → /lib/systemd/system/snapd.core-fixup.service.\n","Created symlink /etc/systemd/system/multi-user.target.wants/snapd.recovery-chooser-trigger.service → /lib/systemd/system/snapd.recovery-chooser-trigger.service.\n","Created symlink /etc/systemd/system/multi-user.target.wants/snapd.seeded.service → /lib/systemd/system/snapd.seeded.service.\n","Created symlink /etc/systemd/system/cloud-final.service.wants/snapd.seeded.service → /lib/systemd/system/snapd.seeded.service.\n","Unit /lib/systemd/system/snapd.seeded.service is added as a dependency to a non-existent unit cloud-final.service.\n","Created symlink /etc/systemd/system/multi-user.target.wants/snapd.service → /lib/systemd/system/snapd.service.\n","Created symlink /etc/systemd/system/timers.target.wants/snapd.snap-repair.timer → /lib/systemd/system/snapd.snap-repair.timer.\n","Created symlink /etc/systemd/system/sockets.target.wants/snapd.socket → /lib/systemd/system/snapd.socket.\n","Created symlink /etc/systemd/system/final.target.wants/snapd.system-shutdown.service → /lib/systemd/system/snapd.system-shutdown.service.\n","Selecting previously unselected package chromium-browser.\n","(Reading database ... 122194 files and directories currently installed.)\n","Preparing to unpack .../chromium-browser_1%3a85.0.4183.83-0ubuntu2.22.04.1_amd64.deb ...\n","=> Installing the chromium snap\n","==> Checking connectivity with the snap store\n","===> System doesn't have a working snapd, skipping\n","Unpacking chromium-browser (1:85.0.4183.83-0ubuntu2.22.04.1) ...\n","Selecting previously unselected package chromium-chromedriver.\n","Preparing to unpack .../chromium-chromedriver_1%3a85.0.4183.83-0ubuntu2.22.04.1_amd64.deb ...\n","Unpacking chromium-chromedriver (1:85.0.4183.83-0ubuntu2.22.04.1) ...\n","Selecting previously unselected package systemd-hwe-hwdb.\n","Preparing to unpack .../systemd-hwe-hwdb_249.11.5_all.deb ...\n","Unpacking systemd-hwe-hwdb (249.11.5) ...\n","Setting up systemd-hwe-hwdb (249.11.5) ...\n","Setting up chromium-browser (1:85.0.4183.83-0ubuntu2.22.04.1) ...\n","update-alternatives: using /usr/bin/chromium-browser to provide /usr/bin/x-www-browser (x-www-browser) in auto mode\n","update-alternatives: using /usr/bin/chromium-browser to provide /usr/bin/gnome-www-browser (gnome-www-browser) in auto mode\n","Setting up chromium-chromedriver (1:85.0.4183.83-0ubuntu2.22.04.1) ...\n","Processing triggers for udev (249.11-0ubuntu3.12) ...\n","Processing triggers for hicolor-icon-theme (0.17-2) ...\n","Processing triggers for libc-bin (2.35-0ubuntu3.4) ...\n","/sbin/ldconfig.real: /usr/local/lib/libtbb.so.12 is not a symbolic link\n","\n","/sbin/ldconfig.real: /usr/local/lib/libtbbbind.so.3 is not a symbolic link\n","\n","/sbin/ldconfig.real: /usr/local/lib/libtbbbind_2_5.so.3 is not a symbolic link\n","\n","/sbin/ldconfig.real: /usr/local/lib/libtbbmalloc.so.2 is not a symbolic link\n","\n","/sbin/ldconfig.real: /usr/local/lib/libtbbbind_2_0.so.3 is not a symbolic link\n","\n","/sbin/ldconfig.real: /usr/local/lib/libtbbmalloc_proxy.so.2 is not a symbolic link\n","\n","Processing triggers for man-db (2.10.2-1) ...\n","Processing triggers for dbus (1.12.20-2ubuntu4.1) ...\n","cp: '/usr/lib/chromium-browser/chromedriver' and '/usr/bin/chromedriver' are the same file\n","Requirement already satisfied: tqdm in /usr/local/lib/python3.10/dist-packages (4.66.2)\n"]}],"source":["#이 부분은 처음 한번만 실행하면 됌.\n","!pip install selenium\n","!apt-get update\n","!apt install chromium-chromedriver\n","!cp /usr/lib/chromium-browser/chromedriver /usr/bin\n","!pip install tqdm"]},{"cell_type":"code","execution_count":2,"metadata":{"id":"hK5oCE65qbe5","executionInfo":{"status":"ok","timestamp":1711495553849,"user_tz":-540,"elapsed":10233,"user":{"displayName":"KyungTae Lim","userId":"06695996952901489598"}}},"outputs":[],"source":["#Colab에선 웹브라우저 창이 뜨지 않으므로 별도 설정한다.\n","# -*- coding: UTF-8 -*-\n","import time\n","from selenium import webdriver\n","from tqdm import tqdm\n","from selenium.webdriver.common.keys import Keys\n","from selenium.webdriver.common.by import By\n","\n","options = webdriver.ChromeOptions()\n","options.add_argument('--headless')        # Head-less 설정\n","options.add_argument('--no-sandbox')\n","options.add_argument('--disable-dev-shm-usage')\n","driver = webdriver.Chrome(options=options)"]},{"cell_type":"code","execution_count":3,"metadata":{"id":"iFOZlSvUMc9i","executionInfo":{"status":"ok","timestamp":1711495562516,"user_tz":-540,"elapsed":4437,"user":{"displayName":"KyungTae Lim","userId":"06695996952901489598"}}},"outputs":[],"source":["#코로나로 검색\n","driver.get('https://search.naver.com/search.naver?query=%EC%BD%94%EB%A1%9C%EB%82%98&where=news&ie=utf8&sm=nws_hty')\n","\n","# 옵션 클릭.\n","elem_option = driver.find_element(By.XPATH, '//*[@id=\"snb\"]/div[1]/div/div[2]/a')\n","elem_option.send_keys('\\n')\n","\n","# # 직접입력 클릭.\n","elem_input = driver.find_element(By.XPATH,'//*[@id=\"snb\"]/div[2]/ul/li[3]/div/div[1]/a[9]')\n","elem_input.send_keys('\\n')\n","\n","naver_news_link = []"]},{"cell_type":"code","execution_count":4,"metadata":{"id":"8WW-wzptMnN9","executionInfo":{"status":"ok","timestamp":1711495589340,"user_tz":-540,"elapsed":444,"user":{"displayName":"KyungTae Lim","userId":"06695996952901489598"}}},"outputs":[],"source":["def select_date_naver_news(year,month,day):\n","\n","    '''네이버뉴스에서 원하는 날짜의 뉴스를 검색해서 네이버뉴스 링크를 리스트로 return.'''\n","    # 연(year) 클릭하기\n","    elem_year = driver.find_element(By.XPATH,f'//*[@id=\"snb\"]/div[2]/ul/li[3]/div/div[3]/div[2]/div[1]/div/div/div/ul/li[{year-1989}]')\n","    elem_year.click()\n","\n","    # 월(month) 클릭하기\n","    elem_month = driver.find_element(By.XPATH,f'//*[@id=\"snb\"]/div[2]/ul/li[3]/div/div[3]/div[2]/div[2]/div/div/div/ul/li[{month}]')\n","    elem_month.click()\n","\n","    # # 날짜(day) 클릭하기\n","    elem_day = driver.find_element(By.XPATH,f'//*[@id=\"snb\"]/div[2]/ul/li[3]/div/div[3]/div[2]/div[3]/div/div/div/ul/li[{day}]')\n","    elem_day.click()\n","\n","    # 끝나는 날짜 클릭하기\n","    driver.find_element(By.XPATH,'//*[@id=\"snb\"]/div[2]/ul/li[3]/div/div[3]/div[1]/span[3]/a').click()\n","\n","    # 연(year) 클릭하기\n","    elem_year = driver.find_element(By.XPATH,f'//*[@id=\"snb\"]/div[2]/ul/li[3]/div/div[3]/div[2]/div[1]/div/div/div/ul/li[{year-1989}]')\n","    elem_year.click()\n","\n","    # 월(month) 클릭하기\n","    elem_month = driver.find_element(By.XPATH,f'//*[@id=\"snb\"]/div[2]/ul/li[3]/div/div[3]/div[2]/div[2]/div/div/div/ul/li[{month}]')\n","    elem_month.click()\n","\n","    # # 날짜(day) 클릭하기\n","    elem_day = driver.find_element(By.XPATH,f'//*[@id=\"snb\"]/div[2]/ul/li[3]/div/div[3]/div[2]/div[3]/div/div/div/ul/li[{day}]')\n","    elem_day.click()\n","\n","    # 적용 클릭\n","    driver.find_element(By.XPATH,'//*[@id=\"snb\"]/div[2]/ul/li[3]/div/div[3]/div[3]/button').click()\n","\n","    naver_news_link = driver.find_elements(By.LINK_TEXT, '네이버뉴스')\n","    links = [link.get_attribute('href') for link in naver_news_link]\n","\n","    # 옵션 초기화 버튼 클릭\n","    driver.find_element(By.XPATH,'//*[@id=\"snb\"]/div[2]/ul/li[7]/div/div/a[1]').click()\n","\n","    time.sleep(2)\n","\n","    # 직접입력 클릭.\n","    elem_input = driver.find_element(By.XPATH,'//*[@id=\"snb\"]/div[2]/ul/li[3]/div/div[1]/a[9]')\n","    elem_input.send_keys(Keys.ENTER)\n","\n","    return links"]},{"cell_type":"code","execution_count":5,"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"Ma6ldRF3Qhtq","outputId":"1bfcf634-99a4-4cc6-f61b-0bce4d8b2167","executionInfo":{"status":"ok","timestamp":1711495648542,"user_tz":-540,"elapsed":25984,"user":{"displayName":"KyungTae Lim","userId":"06695996952901489598"}}},"outputs":[{"output_type":"stream","name":"stderr","text":["preprocessing year:   0%|          | 0/1 [00:00<?, ?it/s]\n","preprocessing month:   0%|          | 0/1 [00:00<?, ?it/s]\u001b[A\n","\n","preprocessing day:   0%|          | 0/31 [00:00<?, ?it/s]\u001b[A\u001b[A\n","\n","preprocessing day:   3%|▎         | 1/31 [00:04<02:12,  4.40s/it]\u001b[A\u001b[A\n","\n","preprocessing day:   6%|▋         | 2/31 [00:08<02:03,  4.26s/it]\u001b[A\u001b[A\n","\n","preprocessing day:  10%|▉         | 3/31 [00:13<02:02,  4.36s/it]\u001b[A\u001b[A\n","\n","preprocessing day:  13%|█▎        | 4/31 [00:17<01:53,  4.20s/it]\u001b[A\u001b[A\n","\n","preprocessing day:  16%|█▌        | 5/31 [00:21<01:47,  4.15s/it]\u001b[A\u001b[A\n","\n","preprocessing day:  19%|█▉        | 6/31 [00:25<01:46,  4.24s/it]\u001b[A\u001b[A\n","\n","preprocessing day:  23%|██▎       | 7/31 [00:25<01:10,  2.94s/it]\u001b[A\u001b[AWARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c317b20>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c317e20>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c317a90>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c350490>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c3508b0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c350b20>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c317ac0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c350640>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c3500d0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c351030>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c3512d0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c351510>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","\n","\n","preprocessing day:  35%|███▌      | 11/31 [00:25<00:20,  1.04s/it]\u001b[A\u001b[AWARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c3516f0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c351c30>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c351ea0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c317b80>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c317f40>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c351ff0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c3515d0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c3510f0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c350be0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c350310>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c3505b0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c3503d0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c3524d0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c352830>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c352a70>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c79f7c0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c350040>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c350a90>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c350b20>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c3503a0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c350220>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c351390>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c351240>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c352050>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c352d10>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c353010>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c317b50>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c3531f0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c351120>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c352b30>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","\n","\n","preprocessing day:  68%|██████▊   | 21/31 [00:25<00:03,  3.13it/s]\u001b[A\u001b[AWARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c351390>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c351180>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c350940>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c350310>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c3522c0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c350a60>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c3533d0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c353610>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c317f10>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c352950>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c350250>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c351780>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c351030>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c350d00>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c351660>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c350e20>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c352140>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c350d60>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c353bb0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c353f10>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c3880d0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c353220>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c350be0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c351750>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c3509a0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c351030>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c350670>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c3521a0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c350c10>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c350b20>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/element\n","preprocessing day: 100%|██████████| 31/31 [00:26<00:00,  1.19it/s]\n","\n","preprocessing month: 100%|██████████| 1/1 [00:26<00:00, 26.02s/it]\n","preprocessing year: 100%|██████████| 1/1 [00:26<00:00, 26.03s/it]\n"]}],"source":["# 2020년 1월부터 ~ 2021 11월 현재까지 2년치 네이버 뉴스데이터 기사 크롤링\n","\n","# 2020년 1월의 네이버 뉴스데이터 기사 크롤링\n","for year in tqdm(range(2024,2024),desc='preprocessing year'):\n","    for month in tqdm(range(1,2),desc='preprocessing month'):\n","        for day in tqdm(range(1,32),desc='preprocessing day'):\n","            try:\n","                res = select_date_naver_news(year=year,month=month,day=day)\n","                naver_news_link.append(res)\n","            except:\n","                pass"]},{"cell_type":"code","source":["res"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"cnIrfBn0GNep","executionInfo":{"status":"ok","timestamp":1711495765702,"user_tz":-540,"elapsed":6,"user":{"displayName":"KyungTae Lim","userId":"06695996952901489598"}},"outputId":"f6a1574c-b9ee-4785-8066-fce15d49938b"},"execution_count":7,"outputs":[{"output_type":"execute_result","data":{"text/plain":["['https://n.news.naver.com/mnews/article/003/0009640953?sid=102',\n"," 'https://n.news.naver.com/mnews/article/421/0004394383?sid=102',\n"," 'https://n.news.naver.com/mnews/article/018/0004549362?sid=104',\n"," 'https://n.news.naver.com/mnews/article/025/0002965713?sid=102',\n"," 'https://n.news.naver.com/mnews/article/015/0004269147?sid=106',\n"," 'https://n.news.naver.com/mnews/article/417/0000489994?sid=106',\n"," 'https://n.news.naver.com/mnews/article/009/0004495421?sid=106',\n"," 'https://n.news.naver.com/mnews/article/057/0001409472?sid=106',\n"," 'https://n.news.naver.com/mnews/article/421/0004393497?sid=102',\n"," 'https://n.news.naver.com/mnews/article/056/0010779813?sid=100',\n"," 'https://n.news.naver.com/mnews/article/055/0000783482?sid=102',\n"," 'https://n.news.naver.com/mnews/article/052/0001385061?sid=106',\n"," 'https://n.news.naver.com/mnews/article/047/0002252368?sid=103']"]},"metadata":{},"execution_count":7}]},{"cell_type":"code","execution_count":6,"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"bFQet1hwKlwQ","outputId":"a0395694-13a9-434c-f195-d9dcf6d5a705","executionInfo":{"status":"ok","timestamp":1711495655225,"user_tz":-540,"elapsed":5,"user":{"displayName":"KyungTae Lim","userId":"06695996952901489598"}}},"outputs":[{"output_type":"execute_result","data":{"text/plain":["[['https://n.news.naver.com/mnews/article/001/0011309413?sid=102',\n","  'https://n.news.naver.com/mnews/article/023/0003497372?sid=110',\n","  'https://n.news.naver.com/mnews/article/421/0004385981?sid=101',\n","  'https://n.news.naver.com/mnews/article/417/0000488359?sid=101',\n","  'https://n.news.naver.com/mnews/article/023/0003497374?sid=110',\n","  'https://n.news.naver.com/mnews/article/023/0003497376?sid=110'],\n"," ['https://n.news.naver.com/mnews/article/421/0004388509?sid=102',\n","  'https://n.news.naver.com/mnews/article/055/0000782596?sid=102',\n","  'https://n.news.naver.com/mnews/article/003/0009635521?sid=101',\n","  'https://n.news.naver.com/mnews/article/056/0010778660?sid=102',\n","  'https://n.news.naver.com/mnews/article/028/0002480290?sid=103',\n","  'https://n.news.naver.com/mnews/article/214/0001004517?sid=101'],\n"," ['https://n.news.naver.com/mnews/article/055/0000782841?sid=102',\n","  'https://n.news.naver.com/mnews/article/020/0003262289?sid=105',\n","  'https://n.news.naver.com/mnews/article/055/0000782926?sid=100',\n","  'https://n.news.naver.com/mnews/article/001/0011313127?sid=104',\n","  'https://n.news.naver.com/mnews/article/421/0004390102?sid=110'],\n"," ['https://n.news.naver.com/mnews/article/023/0003498046?sid=110'],\n"," ['https://n.news.naver.com/mnews/article/001/0011315797?sid=103',\n","  'https://n.news.naver.com/mnews/article/003/0009639248?sid=103',\n","  'https://n.news.naver.com/mnews/article/421/0004392517?sid=106',\n","  'https://n.news.naver.com/mnews/article/009/0004494659?sid=106',\n","  'https://n.news.naver.com/mnews/article/382/0000788657?sid=106',\n","  'https://n.news.naver.com/mnews/article/018/0004549188?sid=104',\n","  'https://n.news.naver.com/mnews/article/584/0000007223?sid=103',\n","  'https://n.news.naver.com/mnews/article/468/0000611669?sid=106',\n","  'https://n.news.naver.com/mnews/article/112/0003253855?sid=106',\n","  'https://n.news.naver.com/mnews/article/117/0003319730?sid=106',\n","  'https://n.news.naver.com/mnews/article/298/0000306652?sid=123',\n","  'https://n.news.naver.com/mnews/article/298/0000306590?sid=123',\n","  'https://n.news.naver.com/mnews/article/298/0000306657?sid=123'],\n"," ['https://n.news.naver.com/mnews/article/003/0009640953?sid=102',\n","  'https://n.news.naver.com/mnews/article/421/0004394383?sid=102',\n","  'https://n.news.naver.com/mnews/article/018/0004549362?sid=104',\n","  'https://n.news.naver.com/mnews/article/025/0002965713?sid=102',\n","  'https://n.news.naver.com/mnews/article/015/0004269147?sid=106',\n","  'https://n.news.naver.com/mnews/article/417/0000489994?sid=106',\n","  'https://n.news.naver.com/mnews/article/009/0004495421?sid=106',\n","  'https://n.news.naver.com/mnews/article/057/0001409472?sid=106',\n","  'https://n.news.naver.com/mnews/article/421/0004393497?sid=102',\n","  'https://n.news.naver.com/mnews/article/056/0010779813?sid=100',\n","  'https://n.news.naver.com/mnews/article/055/0000783482?sid=102',\n","  'https://n.news.naver.com/mnews/article/052/0001385061?sid=106',\n","  'https://n.news.naver.com/mnews/article/047/0002252368?sid=103']]"]},"metadata":{},"execution_count":6}],"source":["naver_news_link"]},{"cell_type":"code","execution_count":8,"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"KKE6R1OzQnBN","outputId":"43dc3e45-4b64-4d91-8fec-2936cbc7e2dc","executionInfo":{"status":"ok","timestamp":1711495860835,"user_tz":-540,"elapsed":13135,"user":{"displayName":"KyungTae Lim","userId":"06695996952901489598"}}},"outputs":[{"output_type":"stream","name":"stderr","text":["\rpreprocessing:   0%|          | 0/6 [00:00<?, ?it/s]WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c0cf580>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c0cf940>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c0cf640>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb214580>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb214820>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb214a30>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c0ceec0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c0cea10>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb214340>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb214f70>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb215210>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb215600>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb215a50>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb215cf0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb215f30>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb216380>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c0ce890>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c0cf640>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n"]},{"output_type":"stream","name":"stdout","text":["Time Out!\n","Time Out!\n","Time Out!\n","Time Out!\n","Time Out!\n","Time Out!\n"]},{"output_type":"stream","name":"stderr","text":["\rpreprocessing:  17%|█▋        | 1/6 [00:02<00:10,  2.06s/it]WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb215c90>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb215930>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb215ae0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb2158a0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb2151b0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb215060>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb214340>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb214940>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb216590>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb216800>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c0cf760>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c0cf6a0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb214c70>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c0cf640>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb216470>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb215db0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb215390>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb215600>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n"]},{"output_type":"stream","name":"stdout","text":["Time Out!\n","Time Out!\n","Time Out!\n","Time Out!\n","Time Out!\n","Time Out!\n"]},{"output_type":"stream","name":"stderr","text":["\rpreprocessing:  33%|███▎      | 2/6 [00:04<00:08,  2.10s/it]WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb215630>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb215180>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb215ba0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb215a20>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb216c80>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb215ed0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c0cf6a0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c0ce8c0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb216e00>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb215180>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb215480>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb215ae0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb215390>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb214220>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb215030>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n"]},{"output_type":"stream","name":"stdout","text":["Time Out!\n","Time Out!\n","Time Out!\n","Time Out!\n","Time Out!\n"]},{"output_type":"stream","name":"stderr","text":["\rpreprocessing:  50%|█████     | 3/6 [00:06<00:06,  2.08s/it]WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb214c70>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb2169b0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb217040>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n"]},{"output_type":"stream","name":"stdout","text":["Time Out!\n"]},{"output_type":"stream","name":"stderr","text":["\rpreprocessing:  67%|██████▋   | 4/6 [00:08<00:04,  2.05s/it]WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb2172e0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c0cf940>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c0ceec0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb216980>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb214c70>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb216590>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb214ee0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb214430>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb2142e0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb2152a0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb214dc0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb215c60>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb217550>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c0ce410>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c0cf580>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb2164a0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb215c60>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb215180>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb215060>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb215d20>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb2151e0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb2170d0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb216fe0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb2171f0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb217a30>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c0cee00>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a031c0cf850>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb2140a0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb216170>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb216fe0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb214c40>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb215780>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb215690>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb215ae0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb215ab0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb214160>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb217f10>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb2158a0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb217c70>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n"]},{"output_type":"stream","name":"stdout","text":["Time Out!\n","Time Out!\n","Time Out!\n","Time Out!\n","Time Out!\n","Time Out!\n","Time Out!\n","Time Out!\n","Time Out!\n","Time Out!\n","Time Out!\n","Time Out!\n","Time Out!\n"]},{"output_type":"stream","name":"stderr","text":["\rpreprocessing:  83%|████████▎ | 5/6 [00:10<00:02,  2.07s/it]WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb215ae0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb214d30>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb214e80>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb217160>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb214b80>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb214340>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb217c40>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb216170>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb216200>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb254190>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb2542b0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb2544c0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb216200>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb2169b0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb216710>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb2143a0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb215c30>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb2152a0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb214430>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb217460>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb215b10>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb216d10>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb217d00>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb254610>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb215090>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb217460>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb216350>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb217ca0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb2169e0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb2140a0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb2178b0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb214220>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb214fa0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb254b20>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb2170a0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb254ee0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb217880>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb2169b0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n","WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7a02fb217430>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/26c3ea6875ccc62ab781d663441170aa/url\n"]},{"output_type":"stream","name":"stdout","text":["Time Out!\n","Time Out!\n","Time Out!\n","Time Out!\n","Time Out!\n","Time Out!\n","Time Out!\n","Time Out!\n","Time Out!\n","Time Out!\n","Time Out!\n","Time Out!\n","Time Out!\n"]},{"output_type":"stream","name":"stderr","text":["preprocessing: 100%|██████████| 6/6 [00:12<00:00,  2.08s/it]\n"]}],"source":["from bs4 import BeautifulSoup\n","from selenium.common.exceptions import UnexpectedAlertPresentException\n","\n","naver_news_title = []\n","naver_news_content = []\n","\n","for n in tqdm(range(len(naver_news_link)),desc='preprocessing'):\n","    news_page_title = []\n","    news_page_content = []\n","\n","    for idx in range(len(naver_news_link[n])):\n","\n","        ######## 긁어온 URL로 접속하기 ########\n","        try:\n","            driver.get(naver_news_link[n][idx])\n","        except:\n","            print(\"Time Out!\")\n","            continue\n","\n","        ######## 접속한 URL에서 page_source받아오기 ########\n","        try:\n","            response = driver.page_source\n","        except UnexpectedAlertPresentException:\n","            driver.switch_to_alert().alert()\n","            print(\"게시글이 삭제된 경우입니다.\")\n","            continue\n","        soup = BeautifulSoup(response,'html.parser')\n","\n","        ##### 뉴스 타이틀 긇어오기 #####\n","        title = None\n","        try:\n","            og_title = soup.find('meta', property='og:title')\n","            title = og_title['content']\n","        except:\n","            title = \"OUT_LINK\"\n","\n","        news_page_title.append(title)\n","\n","        ###### 뉴스 본문 긇어오기 #####\n","        doc = None\n","        text = ''\n","\n","        data = soup.find_all(\"div\",{\"class\":\"newsct_article _article_body\"})\n","        if data:\n","            for item in data:\n","                if item.find('script',type='text/javascript'):\n","                    item.find('script',type='text/javascript').decompose()\n","                elif item.find('span','end_photo_org'):\n","                    item.find('span','end_photo_org').decompose()\n","                text = item.get_text()\n","                doc = ''.join(text)\n","        else:\n","            doc = \"OUT_LINK\"\n","\n","        news_page_content.append(doc.replace('\\n',' '))\n","\n","    naver_news_title.append(news_page_title)\n","    naver_news_content.append(news_page_content)\n","\n","    time.sleep(2)"]},{"cell_type":"code","source":["# 2020년 1월 1일 뉴스\n","naver_news_title[0]"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"pQtF3C3Ofa6F","outputId":"31bda1d3-1d31-4cb9-df62-b94e46f7f0c1","executionInfo":{"status":"ok","timestamp":1711495870426,"user_tz":-540,"elapsed":460,"user":{"displayName":"KyungTae Lim","userId":"06695996952901489598"}}},"execution_count":9,"outputs":[{"output_type":"execute_result","data":{"text/plain":["[]"]},"metadata":{},"execution_count":9}]},{"cell_type":"code","execution_count":10,"metadata":{"colab":{"base_uri":"https://localhost:8080/","height":162},"id":"t8yJkpCZb89a","outputId":"a99f1b0b-7d68-4d51-fa6a-e3746d8e0f80","executionInfo":{"status":"error","timestamp":1711495872744,"user_tz":-540,"elapsed":566,"user":{"displayName":"KyungTae Lim","userId":"06695996952901489598"}}},"outputs":[{"output_type":"error","ename":"IndexError","evalue":"list index out of range","traceback":["\u001b[0;31m---------------------------------------------------------------------------\u001b[0m","\u001b[0;31mIndexError\u001b[0m                                Traceback (most recent call last)","\u001b[0;32m<ipython-input-10-75c0900d69df>\u001b[0m in \u001b[0;36m<cell line: 2>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0;31m# 2020년 1월 8일 뉴스\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 2\u001b[0;31m \u001b[0mnaver_news_title\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;36m7\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m","\u001b[0;31mIndexError\u001b[0m: list index out of range"]}],"source":["# 2020년 1월 8일 뉴스\n","naver_news_title[7]"]},{"cell_type":"code","execution_count":null,"metadata":{"colab":{"base_uri":"https://localhost:8080/","height":226},"id":"Yn2VpZeXSPdw","outputId":"6f354c3c-adf0-458b-ac4c-97846e461f74"},"outputs":[{"output_type":"execute_result","data":{"text/plain":["'  질병관리본부 \"격리치료 및 검사…현재 상태 양호\"\\'전파 증거 없다\\'는 중국 발표 따라 \\'관심단계\\' 유지[세종=뉴시스] 임재희 기자 = 질병관리본부는 중국 우한시를 방문했다가 지난달 30일 입국한 중국 국적의 36세 여성 1명이 원인불명 폐렴 증상을 보여 격리 치료 중이라고 8일 밝혔다.질병관리본부 관계자는 \"조사대상 유증상자로 분류해 격리 치료 및 검사를 실시 중\"이라며 \"상기 환자는 화난 해산물시장 방문이나 야생동물 접촉은 없었고 현재 상태는 양호하다\"고 말했다.환자는 업무차 지난달 13~17일 중국 우한시를 방문했으며 한국에는 같은달 30일 입국했다. 기초 역학조사 결과 다음날인 31일부터 기침, 목 붓는 증상이 처음 나타났고 이달 7일 한림대학교 동탄성심병원에서 진료 중 우한시 방문력과 폐렴 소견이 확인, 질병관리본부에 신고됐다.중국 우한시 위생건강위원회가 지난달 31일 원인불명 폐렴 사실을 최초 발표한 이후 이달 5일 현재까지 59명의 환자가 보고됐다.질병관리본부는 우한시 방문력과 흉부방사선검사상 폐렴 소견을 근거로 이번 사례를 \\'조사대상 유증상자\\'로 분류했고 환자를 국가지정입원 치료병상이 있는 분당서울대병원으로 이송해 격리 상태로 치료와 검사를 했다. 현재 중앙 및 경기도 역학조사관이 역학조사를 진행 중이다.해당 환자는 이달 2~3일 기침과 발열로 오산한국병원에서 흉부방사선 검사 정상 판정에 따라 감기약 처방을 받았으며 6일에는 동탄성심병원에서 흉부방사선 검사 정상, 계절인플루엔자 검사 음성 판정을 받은 것으로 확인됐다. 해당 의료인에 대해선 진료 업무 배제 및 진료 공간 폐쇄 등 조치는 하지 않은 상태다.경기도 보건환경연구원이 시행한 호흡기바이러스 9종 검사결과는 음성이었으며 질병관리본부가 추가적으로 폐렴 유발 원인병원체에 대한 검사를 진행 중이다.메르스, 인플루엔자, 파라인플루엔자, 아데노바이러스, 사람보카바이러스, 호흡기세포융합바이러스, 리노바이러스, 사람메타뉴모바이러스, 사람코로나바이러스 등은 아니라는 얘기다.동반 여행자 및 접촉자는 조사 중으로 접촉자는 해당 보건소가 발병 여부를 모니터링할 예정이다.이와 관련해 질병관리본부는 8일 위기평가회의를 열었으나 관심 단계(해외에서의 신종감염병의 발생 및 유행)를 유지하기로 했다. 병원체 검사와 역학조사가 진행 중인 점과 사람간 전파 및 의료인 감염의 증거가 아직 없다는 중국 보건당국의 발표를 근거로 한 결정이다. 대신 예방관리대책을 한층 강화하기로 하고 추후에 중국 등 발생상황을 예의주시하며 조사대상 유증상자의 검사 및 역학조사결과를 바탕으로 위기단계 조정을 검토할 예정이다.앞서 질병관리본부는 3일부터 \\'우한시 원인불명 폐렴 대책반\\'을 가동하고 긴급상황실 24시간 대응체계를 운영 중이다. 중국 우한시 원인불명 폐렴은 원인병원체가 확인되기 전까지 \\'신종감염병증후군\\'으로 간주해 검역, 격리, 역학조사 등을 시행하고 있다.출입국 관리 강화를 위해 우한시 입국자를 대상으로 개인별 발열 감시, 건강상태질문서 징구, 주의안내문(출국자 및 입국자 대상) 배부 등 검역을 강화하고 유증상자 검역조사를 하고 있다. 일주일에 우한시와 인천공항을 오가는 직항 항공기는 8편이다.우한시 원인불명 폐렴 대응절차를 상세히 마련해 의료기관 대상으로 배포하고 의사협회와 협력하여 전체 의사에게 중국유행상황, 신고안내 문자를 발송하여 환자 감시체계를 가동하는 등 지역사회 감시체계를 가동 중이다.우한시 입국자 정보는 9일 DUR(의약품안전사용서비스)를 통해 의료기관에 제공해 내원환자 우한시 방문력을 확인 가능하도록 조치할 예정이다.질병관리본부는 우한시 방문객들은 가금류나 야생동물 접촉을 피하고 아픈 사람(발열, 기침 등 호흡기감염 증상)과 접촉을 피하며 해외여행 시에는 손 씻기, 기침예절 등 개인위생수칙을 준수하고 귀국 후 14일 이내 증상 발생 시 질병관리본부 콜센터(1339)로 상담해 줄 것을 당부했다.☞공감언론 뉴시스 limj@newsis.com▶ K-Artprice 모바일 오픈! 미술작품 가격을 공개합니다▶ 뉴시스 채널 구독하고 에어팟 프로 받아가세요▶ 뉴시스 빅데이터 MSI 주가시세표 바로가기 <저작권자ⓒ 공감언론 뉴시스통신사. 무단전재-재배포 금지.> \\t\\t '"],"application/vnd.google.colaboratory.intrinsic+json":{"type":"string"}},"metadata":{},"execution_count":55}],"source":["naver_news_content[7][1]"]},{"cell_type":"code","execution_count":null,"metadata":{"id":"wQFhZDpTRNSx"},"outputs":[],"source":["import pickle\n","\n","# 크롤링된 결과 저장\n","with open(\"data\\kr_naver_news_title.pk\",'wb') as f:\n","    pickle.dump(naver_news_title,f)\n","with open(\"data\\kr_naver_news_content.pk\",'wb') as f:\n","    pickle.dump(naver_news_content,f)"]},{"cell_type":"code","source":[],"metadata":{"id":"pAtSJEPwfqnW"},"execution_count":null,"outputs":[]}],"metadata":{"colab":{"provenance":[]},"kernelspec":{"display_name":"Python 3","name":"python3"},"language_info":{"name":"python"}},"nbformat":4,"nbformat_minor":0}